client/src/assets/images/icons/index.js

```diff
@@ -210,3 +210,12 @@ export const ArrowUpIcon = () => (
     />
   </Icon>
 );
+
+export const SearchIcon = () => (
+  <Icon data-icon="arrow-up" viewBox="0 0 512 512">
```

Comment:
data-icon is incorrect. What is this prop for anyway?

Reply:
it is here just for accessibility reasons. 
Change it to`dataIcon` or remove it anyway?

Reply:
"arrow-up" should be "search"